src/mplhep/plot.py

```diff
@@ -392,17 +429,45 @@ def iterable_not_string(arg):
 
     # Add sticky edges for autoscale
     _artist.sticky_edges.y.append(0)
-
     if xtick_labels is None:
         if binticks:
             _slice = int(round(float(len(final_bins)) / len(ax.get_xticks()))) + 1
             ax.set_xticks(final_bins[::_slice])
+    elif flow == "show":
+        if binticks:
+            _slice = int(round(float(len(final_bins)) / len(ax.get_xticks()))) + 1
+            ax.set_xticks(final_bins[::_slice])
     else:
         ax.set_xticks(_bin_centers)
         ax.set_xticklabels(xtick_labels)
 
     if x_axes_label:
         ax.set_xlabel(x_axes_label)
+    if flow == "hint":
```

Comment:
I was thinking more along the lines of an arrow maybe? I am also not sure this should be printing (for example if one is making a batch of plots, this might just spam the console)

Reply:
As we already have option to display, then I just remove printing